dts/bindings/clock/espressif,esp32-cam-clk.yaml

```diff
@@ -0,0 +1,21 @@
+#
+# Copyright (c) 2019, Linaro Limited
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+description: NXP MCUX CMOS sensor interface
+
+compatible: "espressif,esp32-cam-clk"
+
+include: [base.yaml, pinctrl-device.yaml]
+
+properties:
```

Comment:
As I mentioned above, some of the definitions hardcoded in the driver could assume a property here.

Reply:
Same as above.